services/tpl.js

```diff
@@ -0,0 +1,14 @@
+var dom = require('@nymag/dom');
+
+/**
+ * grab templates from the dom
+ * @param {string} selector
+ * @returns {Element}
+ */
+function getTemplate(selector) {
+  var template = dom.find(selector);
+
+  return document.importNode(template.content, true);
+}
+
+module.exports.get = getTemplate;
```

Comment:
todo: add some tests for `getTemplate`, just to prove it works on all supported browsers